src/Parser/Core/CombatLogParser.js

```diff
@@ -269,6 +271,9 @@ class CombatLogParser {
 
     infernalCinders: InfernalCinders,
     umbralMoonglaives: UmbralMoonglaives,
+
+    //added last so that it includes items that are added to the spellbook on initialization.
+    deathRecapTracker: DeathRecapTracker,
```

Comment:
This doesn’t work like this. Import order is determined by dependencies, anything with a single dependency not loaded will be loaded on a later iteration.

Reply:
The DeathRecapTracker sets the list on IDs to track in it's 'on_initialize'. There's probably a better way to handle it but I didn't want to mess around too much with it, and it seemed to consistently not include Healthstone if the Healthstone module was after it in the dependency list, and likewise always include it when HS module was before it.

Reply:
Okay I'm dumb, just added healthstone as a dependency to the death recap tracker <.<